modules/pubmaticBidAdapter.js

```diff
@@ -1106,6 +1126,9 @@ export const spec = {
       if (bid.params.hasOwnProperty('bcat') && isArray(bid.params.bcat)) {
         blockedIabCategories = blockedIabCategories.concat(bid.params.bcat);
       }
+      if (bid.params.hasOwnProperty('acat') && isArray(bid.params.acat)) {
```

Comment:
https://github.com/prebid/Prebid.js/issues/7925 please consider the ortb2 config object as well and document which you will consider first and which you will fallback to if both are set

Reply:
Changes done, considering ortb2 config as well.